src/ServicePulse.Host/vue/src/components/configuration/HealthCheckNotifications_ConfigureEmail.vue

```diff
@@ -98,7 +106,7 @@ function close() {
               </div>
               <div class="row"></div>
               <div class="form-group" :class="{ 'has-error': !toIsValid }">
-                <label for="to">To address</label>
+                  <label for="to">To address <br/>(Separate multiple email address with a semicolon. E.g. testing@test.com,blahbla@bla.com)</label>
```

Comment:
```suggestion
                  <label for="to">To address <br/>(Separate multiple email address with a comma. E.g. jack@skellington.com,oogie@boogie.com</label>
```

It's a [comma](https://learn.microsoft.com/en-us/dotnet/api/system.net.mail.mailmessage.-ctor?view=net-7.0#system-net-mail-mailmessage-ctor(system-string-system-string)), and blahbla doesn't seem like a good example.